qemu/tests/hugepage_specify_node.py

```diff
@@ -0,0 +1,53 @@
+import logging
+
+from virttest import env_process
+from virttest import error_context
+from virttest import utils_misc
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Qemu allocate hugepage from node.
+    Steps:
+    1) System setup hugepage on host.
+    2) Mount this hugepage to /mnt/kvm_hugepage.
+    3) Boot guest only allocate hugepage from node 0.
```

Comment:
The NUMA node should allow the user to specify

Reply:
correct